marcnoon-lawndale

```diff
@@ -0,0 +1,11 @@
+# Lawndale, CA
```

Comment:
@marcnoon: I'll be in CA next week! What part is Lawndale in? How is the weather this time of year?